main.tf

```diff
@@ -1,46 +1,68 @@
 locals {
-  datacenters     = join(",", var.nomad_datacenters)
-  buckets         = var.hive_bucket # output variable for presto
-  hive_env_vars   = join("\n",
+  datacenters   = join(",", var.nomad_datacenters)
+  buckets       = var.hive_bucket # output variable for presto
+  hive_env_vars = join("\n",
     concat([
       "JUST_EXAMPLE_ENV=some-value",
     ], var.hive_container_environment_variables)
   )
+  vault_provider = var.postgres_vault_secret.use_vault_provider || var.minio_vault_secret.use_vault_provider
+  vault_kv_policy_name = jsonencode(
+    concat(
+      [var.postgres_vault_secret.vault_kv_policy_name],
+      [var.minio_vault_secret.vault_kv_policy_name]
+    )
```

Comment:
What will this render as a list?
```
vault_kv_policy_name = [
  "secret/data/random-string/minio",
  "secret/data/random-string/postgres"
]
```

Reply:
It is important to use `jsonencode`, because it will returm `["policy1", "policy2"]` value. And than we can ender it as it is, due to [`vault.policies` stanza](https://www.nomadproject.io/docs/job-specification/vault#policies)

<img width="975" alt="Screenshot 2020-11-10 at 08 43 49" src="https://user-images.githubusercontent.com/15572799/98644558-f32ac180-2330-11eb-9a5c-3004e27362e1.png">

Reply:
All right! Thanks 👌